src/EventSubscriber/RubricChosenSubscriber.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace App\EventSubscriber;
+
+
+use App\Model\SearchData;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+class RubricChosenSubscriber implements EventSubscriberInterface
```

Comment:
Persönlich fände ich `ChosenRubricSubscriber` besser als `RubricChosenSubscriber`, oder was meinst Du?
```suggestion
class ChosenRubricSubscriber implements EventSubscriberInterface
```